src/modules/font.c

```diff
@@ -82,6 +87,9 @@ FONT_RASTER_print(WrenVM* vm) {
   unsigned char *bitmap;
   int w, h;
 
+  int fontHeight = raster->height;
+  int newlines = 0;
```

Comment:
We shouldn't need a newLines variable here, for the same reason as last time.

Reply:
In this case for rasterized fonts the height can't be added directly to `posY` since they are calculated further down, with variables initialized later. This could be done if the `bitmap` was calculated even though we run into newlines, but that would be a waste of performance since 4 variables are initialized and two functions are called which don't need to be called.

Reply:
I believe in these case we would actually modify the value of the `baseY` rather than the `posY` value, as that is the Y position for the baseline of the text.

Reply:
Ah yeah that's true, I didn't even consider that.